common_design.libraries.yml

```diff
@@ -239,6 +239,15 @@ cd-date:
   dependencies:
     - common_design/css-vars-ponyfill
 
+cd-disclosure:
+  css:
+    component:
+      components/cd-disclosure/cd-disclosure.css: {}
+    state:
+      components/cd-disclosure/cd-disclosure-states.css: {}
```

Comment:
I guess another high-level question: worth it to have this "states" file or combine into one component file? Drupal aggregates differently based on this libraries config, so I wanted to give it a spin. But if it seems unweildy, I'm happy to recombine.

Reply:
Looking at the number of rules (especially, see other comment about them), I don't think it's worthy to have 2 css files.